src/makeCalculator.js

```diff
@@ -4,7 +4,28 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+    add: (num) => calculator.result + num,
+    subtract: (num) => calculator.result - num,
+    multiply: (num) => calculator.result * num,
+    divide: (num) =>
+      num !== 0 ? calculator.result / num : 'Error: Division by zero',
```

Comment:
[CODE STYLE] It's recommended to use arithmetic operators with assignment operator `=`. So, instead of `calculator.result / num`, you should write `calculator.result /= num`. Also, remember to handle the case when `num` is 0 to avoid division by zero.